packages/markdown-preview/lib/main.js

```diff
@@ -12,10 +12,18 @@ const isMarkdownPreviewView = function (object) {
 }
 
 module.exports = {
-  activate () {
+  activate() {
     this.disposables = new CompositeDisposable()
     this.commandSubscriptions = new CompositeDisposable()
 
+    this.style = new CSSStyleSheet()
+
+    // When we upgrade Electron, we can push onto `adoptedStyleSheets`
```

Comment:
Can you mark this as `TODO`? This way we can easily find things to change in the future :)

```suggestion
    // TODO: When we upgrade Electron, we can push onto `adoptedStyleSheets`
```